docker/pdo_ccf.dockerfile

```diff
@@ -20,9 +20,9 @@ FROM pdo_ccf_base:${PDO_VERSION}
 # -----------------------------------------------------------------
 # set up the PDO sources
 # -----------------------------------------------------------------
-ARG REBUILD 0
+ARG REBUILD=0
 
```

Comment:
ok.. for consistency i'm fine with this. but... turns out that "=" is optional.

Reply:
i know it's optional, just did it for consistency ...